fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/call/CallFunc.java

```diff
@@ -28,13 +29,13 @@ public abstract class CallFunc {
     /**
      * Get the instance of CallFunc
      */
-    public static CallFunc getFunc(UserIdentity user, UnboundFunction unboundFunction) {
+    public static CallFunc getFunc(ConnectContext ctx, UserIdentity user, UnboundFunction unboundFunction) {
         String funcName = unboundFunction.getName().toUpperCase();
         switch (funcName) {
-            case "EXECUTE_STMT":
+            case "EXECUTE_STMT": // Call built-in functions first
                 return CallExecuteStmtFunc.create(user, unboundFunction.getArguments());
             default:
-                throw new IllegalArgumentException("unknown function name: " + funcName);
+                return CallProcedure.create(ctx, unboundFunction.getSource());
```

Comment:
Should throw exception if user created a store procedure named EXECUTE_STMT.

Reply:
Added a TODO, built-in functions require a separate management